dozer-log-python/src/mapper.rs

```diff
@@ -76,13 +80,37 @@ fn map_value(value: Field, py: Python) -> PyResult<Py<PyAny>> {
         Field::Decimal(v) => Ok(v.to_string().to_object(py)),
         Field::Timestamp(v) => Ok(v.to_string().to_object(py)),
         Field::Date(v) => Ok(v.to_string().to_object(py)),
-        Field::Bson(v) => Ok(v.to_object(py)),
+        Field::Json(v) => map_json_py(v, py),
         Field::Point(v) => map_point(v, py),
         Field::Duration(v) => Ok(v.to_string().to_object(py)),
         Field::Null => Ok(py.None()),
     }
 }
 
+fn map_json_py(val: JsonValue, py: Python) -> PyResult<Py<PyAny>> {
+    match val {
+        JsonValue::Null => Ok(py.None()),
+        JsonValue::Bool(b) => Ok(b.to_object(py)),
+        JsonValue::Number(n) => Ok(n.to_object(py)),
+        JsonValue::String(s) => Ok(s.to_object(py)),
+        JsonValue::Array(a) => Ok(PyList::new(
+            py,
+            a.into_iter().map(|val| map_json_py(val, py).unwrap()),
+        )
+        .to_object(py)),
+        JsonValue::Object(o) => {
+            let obj = PyDict::new(py);
+            for (key, val) in o {
+                obj.set_item(key, map_json_py(val, py).unwrap()).map_or(
```

Comment:
Use `?` instead of `unwrap`.

Can we also use `?` instead of `map_or`?

Reply:
right right fixed!